front/pages/api/apps/[user]/[sId]/runs/index.ts

```diff
@@ -8,16 +8,19 @@ import withLogging from "@app/logger/withlogging";
 import { RunRunType, RunType } from "@app/types/run";
 import { NextApiRequest, NextApiResponse } from "next";
 
-export type GetRunsResponseBody = {
-  runs: RunType[];
-  total: number;
-};
-
-export type PostRunsResponseBody =
+export type GetRunsResponseBody =
   | {
-      run: RunType;
+      runs: RunType[];
+      total: number;
     }
-  | ErrorResponse;
+  | {
```

Comment:
went with this in the end @spolu, since returning `APIError` means updating client code. Happy to do that too, but prefer to do it in a separate PR as it's kind of unrelated to adding TS bindings for DustAPI